docusaurus/docs-internals/editor-controls/data-controls/fileUpload.md

```diff
@@ -0,0 +1,177 @@
+---
+toc_max_heading_level: 4
+---
+# FileUpload
+
+The `fileUpload` control displays a file chooser section. It allows users to select a file from their PC.
```

Comment:
The fileUpload control displays a file chooser section that allows the user to choose one or more files from their device storage.